### PR TITLE
Speed up subscriptions load time

### PR DESCRIPTION
**Pull request by ms-mgrant**

### CODE REVIEW

File: src/SaaS.SDK.CustomerProvisioning/Controllers/HomeController.cs [hunk 1]
```diff
@@ -300,9 +300,10 @@ public IActionResult Subscriptions()
                     this.TempData["ShowWelcomeScreen"] = "True";
                     SubscriptionViewModel subscriptionDetail = new SubscriptionViewModel();
                     subscriptionDetail.Subscriptions = this.subscriptionService.GetPartnerSubscription(this.CurrentUserEmailAddress, default, true).ToList();
+                    var allPlans = this.planRepository.Get().ToList();
```
Comment (santhoshb-msft) [inline src/SaaS.SDK.CustomerProvisioning/Controllers/HomeController.cs:303]: Can we not have the PlanId return when fetching subscriptions from the DB

Comment (ms-mgrant) [reply src/SaaS.SDK.CustomerProvisioning/Controllers/HomeController.cs:303]: If I restricted the allPlans to only fetch plans belonging to the user (in plan repo), would that solve the issue?

Comment (ms-mgrant) [reply src/SaaS.SDK.CustomerProvisioning/Controllers/HomeController.cs:303]: Would using the PlanGUId work better in this case?

**Comment by code4clouds** [comment]
@ms-mgrant I would recommend that for new features that a screenshot of the changes is included. It just makes easy to understand the impact of the changes. ;-)

File: src/SaaS.SDK.CustomerProvisioning/Controllers/HomeController.cs (hunk 1 above)
Comment (santhoshb-msft) [reply src/SaaS.SDK.CustomerProvisioning/Controllers/HomeController.cs]: True but we have to wait till the PlanGuidId change is merged.

**Pull request closed without merging**